Services/CRUD/CrudManager.php

```diff
@@ -225,6 +227,29 @@ public function persist(CrudEntityInterface $entity)
     }
 
     /**
+     * Create one object
+     *
+     * @param CrudEntityInterface $object Object to create
+     *
+     * @throws ValidationFailedException
+     * @return void
+     */
+    public function createOne(CrudEntityInterface $object)
+    {
+        $this->create($object, false);
+        if ($object instanceof ValidatableInterface) {
```

Comment:
Maybe its better to improve `save` method and move check of ValidatableInterface there? We are already throwing `ValidationFailedException` in `save` method, so we can combine them and flush is also there.